packages/patterns/src/index.ts

```diff
@@ -3,12 +3,12 @@
 export * from "./components/actionBar/ActionBar";
 export * from "./components/advancedPanel/AdvancedPanel";
 export * from "./components/anchorNav/AnchorNav";
-export * from "./components/CardGrid";
+export * from "./components/cardGrid/CardGrid";
```

Comment:
This is giving me this error on runtime:
`Cannot find module './components/cardGrid/CardGrid' or its corresponding type declarations.`

Looks like it _should_ be `CardGrid` 🤔